docs/INSTALLING.md

```diff
@@ -0,0 +1,71 @@
+# How do I install the Code Corps Ember app?
+
+### Requirements
+
+You will need to install the following: 
+
+- [Node.js/npm](https://docs.npmjs.com/getting-started/installing-node) we use the LTS version (v 4.4.7)
```

Comment:
4.5 does seem to be the one in circle and as ember specifies. I'll just update it to read "we use the LTS branch" so that we don't need to update this every time ember upgrades its compatability.